### PR TITLE
Fixes postgres configuration for bloomberg

### PR DESCRIPTION
**Pull request by acarrera42**
Fixes postgres configuration for bloom berg's unit test setup.

### CODE REVIEW

**Comment by acarrera42** [comment]
@vishalya @gfiorav Can you please review?  Thanks.

**Comment by acarrera42** [comment]
This fixes RDSIMAPS-3166, RDSIMAPS-3167, RDSIMAPS-3168

File: services/table-geocoder/spec/factories/database.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "host"      : "localhost",
-  "port"      : 5432,
+  "port"      : 4302,
```
Comment (gfiorav) [inline services/table-geocoder/spec/factories/database.json:3]: Where is this being used and what error is rising?

Maybe this values shouldn't be used at all, that's why I'm asking

**Review by gfiorav — COMMENTED** [review]
Made a comment, but if I'm out by the time you respond then go for it!

File: services/table-geocoder/spec/factories/database.json (hunk 1 above)
Comment (acarrera42) [reply services/table-geocoder/spec/factories/database.json:3]: This appears to be used in the db connection for /services/table-geocoder/spec/factories/pg_connection.rb

**Comment by acarrera42** [comment]
This also fixes RDSIMAPS-3156

**Comment by acarrera42** [comment]
This also fixes RDSIMAPS-3157

**Pull request merged**